lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -129,13 +129,10 @@ export default function LMSGrader({
         )}
       >
         <div>
-          <h1 className="text-lg p-0 m-0" data-testid="assignment-name">
+          <h1 className="text-lg font-bold" data-testid="assignment-name">
```

Comment:
We no longer need to suppress margins or padding but we do need to bold this heading. It already had size via `text-lg`.